//#ifndef LOGOVIEW_H
//#define LOGOVIEW_H

//#include <QMainWindow>
//#include <QSqlDatabase>
//#include <QStandardItemModel>
//#include "dao/logomanager.h"

//namespace Ui {
//class logoView;
//}

//class logoView : public QMainWindow
//{
//    Q_OBJECT

//public:
//    explicit logoView(QWidget *parent = nullptr);
//    ~logoView();

//    void showDataList(const int  type);

////定义槽函数
//public slots:
//    void searchBtnSlot();
//    void preBtnSlot();
//    void nextBtnSlot();
//private:
//    Ui::logoView *ui;
//    // 当界面构建之后需要初始化成员
//    //管理数据的成员
//    logoManager * teacherManager;
//    //数据条目渲染的成员 qt
//    QStandardItemModel *standardItemModel;

//};

//#endif // LOGOVIEW_H
